frontend/demo/BUILD

```diff
@@ -8,6 +8,11 @@ module_deps = [
 ts_library(
     name = "app",
     deps = module_deps + [
+        "@npm//@airyhq/components",
```

Comment:
I thought the demo is supposed to depend on the local version of the library. Is there a reason to change this?

Reply:
the showcase of components yes, not the demo. But maybe I understood it the wrong way @lucapette

Reply:
it's a choice so there's no right answer. What I like about this approach: we'd use the library as any other npm package so we'd effectively be users of our own API

Reply:
that is what I thought. Then we have the components show case using the bazel library so we can also test changes before publishing new version to npm

Reply:
Thanks for clearing that up 👍 Then the confusion was on my part.